website/docs/user_guide/activation-strategies.md

```diff
@@ -5,7 +5,11 @@ title: Activation Strategies
 
 It is powerful to be able to turn a feature on and off instantaneously, without redeploying the application. The next level of control comes when you are able to enable a feature for specific users or enable it for a small subset of users. We achieve this level of control with the help of activation strategies. The most straightforward strategy is the standard strategy, which basically means that the feature should be enabled to everyone.
 
-The definition of an activation strategy lives in the Unleash API and can be created via the Unleash UI. The implementation of activation strategies lives in various client implementations.
+Unleash comes with a number of built-in strategies (described below) and also lets you add your own [custom activation strategies](../advanced/custom-activation-strategy.md) if you need more control.
+However, while activation strategies are *defined* on the server, the server does not *implement* the strategies. Instead, activation strategy *implementation* (and thus feature toggle *evaluation*) is done client-side.
+Thus, all [server-side client SDKs](../sdks/index.md#server-side-sdks) and the [Unleash Proxy](../sdks/unleash-proxy.md) implement the default strategies
+(and allow you to add your own custom strategy implementations).
```

Comment:
Is this line break here intentional?

Reply:
Nah, that one wasn't actually intentional; good catch! That said, it doesn't change the output in any way, so it doesn't matter much. I'll fix it up!

Reply:
Yeah, it's perfect, just me and my white space haha